opencog/nlp/scm/nlp.scm

```diff
@@ -7,6 +7,29 @@
 (define relex-server-host "127.0.0.1")
 (define relex-server-port 4444)
```

Comment:
it probably would have been enough to just say `define-public` above, right?  That way, you would not need to create wrapper routines?